dvc/exceptions.py

```diff
@@ -256,7 +256,9 @@ def __init__(self, amount):
 
 
 class CheckoutError(DvcException):
-    def __init__(self, target_infos):
+    def __init__(self, target_infos, stats=None):
+        self.target_infos = target_infos
+        self.stats = stats
         targets = [str(t) for t in target_infos]
         m = (
             "Checkout failed for following targets:\n {}\nDid you "
```

Comment:
sorry, really minor request (feel free to nor change this), came out as I was reviewing:

```
ERROR: unexpected error - Checkout failed for following targets:
 data/prepared
data/features
data/data.xml
```

there is an extra space that is not needed before `data/prepared`. I think the change is `\n {}\n` -> `\n{}\n`